webpack.config.js

```diff
@@ -37,14 +37,11 @@ module.exports = {
     rules: [
       {
         test: /\.(png|jpg|gif)$/i,
-        use: [
-          {
-            loader: "url-loader",
-            options: {
-              limit: 8192,
-            },
-          },
-        ],
+        type: "asset",
```

Comment:
Seeing as we're using Webpack 5's asset loader for .plutus files, I thought we might as well switch for images too. "asset" has the same behaviour as "url-loader" with limit 8kb